# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/reftest.list
# Commit: ea30c0eca521
# Full Hash: ea30c0eca521cf3ffc2fcc60a3b5cd547d8b0500
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2025-05-07 04:19:47
# Regressor Bug: 1961387
# File Overlap Count: 3
# Description:
#   Bug 1961387 - Add a reftest with larger glyphs, for CBDT version of Noto Color Emoji. r=gfx-reviewers,lsalzman
#   
#   Without the scaling fix, the advance of the bitmap emoji glyph isn't
#   scaled according to the rendered size, and so the large glyph extends
#   beyond the expected rect.
# ==============================================================================

diff -r 057a31f469ac -r ea30c0eca521 gfx/tests/reftest/reftest.list
--- a/gfx/tests/reftest/reftest.list	Tue May 06 20:12:46 2025 +0000
+++ b/gfx/tests/reftest/reftest.list	Tue May 06 20:12:46 2025 +0000
@@ -49,4 +49,5 @@
 == 1909933.html 1909933-ref.html
 fuzzy(0-2,0-6229) random-if(Android) == 1912431-emoji-globalAlpha.html 1912431-emoji-globalAlpha-ref.html
 == 1961387-1.html 1961387-1-ref.html
+== 1961387-2.html 1961387-2-ref.html
 skip-if(!cocoaWidget) == 1964225-1.html 1964225-1-ref.html
